### ISSUE TITLE
what is the replacement for azure-sdk-for-go/services/graphrbac/1.6/graphrbac package?

### ISSUE THREAD

**Issue by pillaimithun** [Azure/azure-sdk-for-go#20660]
### Bug Report

- We have some modules using azure-sdk-for-go/services/graphrbac/1.6/graphrbac
- Is there any new library which can be used to create AD user and ApplicationClients inside azure-sdk-for-go/sdk/resourcemanager which replaces graphrbac.ApplicationsClient
- Cannot see the services folder in the latest https://github.com/Azure/azure-sdk-for-go
- If so can you provide an example. Cannot see anything in he samples repo too.

**Comment by jhendrixMSFT** [comment]
The graphrbac package is based on the older Azure AD graph endpoints which have been [deprecated](https://azure.microsoft.com/en-us/updates/update-your-apps-to-use-microsoft-graph-before-30-june-2022/) and replaced by MSGraph.  You can find the MSGraph Go SDK [here](https://github.com/microsoftgraph/msgraph-sdk-go).

**Comment by pillaimithun** [comment]
Thanks for the answer @jhendrixMSFT 

As a follow up question , let me rephrase the question a bit.
For App Registration in AD I used to use azure-sdk-for-go/services/graphrbac/1.6/graphrbac/graphrbacapi.applications.go
and we used users.go for creating new AD users.Looks like its deprecated now.

But readme of below  new [sdk](https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/resourcemanager/aad/armaad) says  "Azure Active Directory Module for Go"
  https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/resourcemanager/aad/armaad
  Can this module be used for App Registration in AD ? If not whats the purpose of this armaad.

Or is it like going forward https://github.com/microsoftgraph/msgraph-sdk-go is the only way we could register an application or create a new AD user?

**Comment by jhendrixMSFT** [comment]
You'll need to use the `msgraph-sdk-go` SDK for this.

The `armaad` module contains APIs for the Microsoft.AADIAM resource provider.  I'm having a hard time finding docs for this, but it doesn't contain APIs for app registration or other MSGraph functionality.

**Comment by pillaimithun** [comment]
> You'll need to use the `msgraph-sdk-go` SDK for this.
> 
> The `armaad` module contains APIs for the Microsoft.AADIAM resource provider. I'm having a hard time finding docs for this, but it doesn't contain APIs for app registration or other MSGraph functionality.


@jhendrixMSFT is https://github.com/microsoftgraph/msgraph-sdk-go is it still under preview can I use it in production?

**Comment by jhendrixMSFT** [comment]
The MSGraph SDK is owned and managed by a separate team so you'll want to open an issue in that repo to find out what their support policies are.

**Comment by pillaimithun** [comment]
Following  up in MSGraph SDK  https://github.com/microsoftgraph/msgraph-sdk-go/issues/474
